checks.d/docker.py

```diff
@@ -16,6 +17,8 @@
 
 EVENT_TYPE = SOURCE_TYPE_NAME = 'docker'
 
+ECS_PORT = '51678/tcp'
```

Comment:
the response looks like this: ```"Ports": {
            "51678/tcp": [
                {
                    "HostIp": "127.0.0.1",
                    "HostPort": "51678"
                }
            ]
        }```
We can either hardcode the port, or choose arbitrary the first port in `ecs_config.get("Ports", {}).keys()`. In both case it may change in the future (the port can change, there can be another port open too). Should I pick the other solution?